src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4751,7 +4751,11 @@ internal static string GetGetVerbText()
 param(
     [Parameter(ValueFromPipeline=$true)]
     [string[]]
-    $verb = '*'
+    $Verb = '*',
+
+    [Parameter(ValueFromPipeline=$true)]
+    [string[]]
+    $Group
```

Comment:
@lzybkr Thanks for great comment! I was asking about using VerbGroups in Issue addressed  in the PR  but there were no examples. So I don't see the need for the public api at all.